colcon_gradle/task/gradle/__init__.py

```diff
@@ -11,6 +12,13 @@
 """Environment variable to override the Gradle executable"""
 GRADLE_COMMAND_ENVIRONMENT_VARIABLE = EnvironmentVariable(
     'GRADLE_COMMAND', 'The full path to the Gradle executable')
+    
+"""Environment variable to override the Gradle executable"""
+GRADLE_HOME_ENVIRONMENT_VARIABLE = EnvironmentVariable(
+    'GRADLE_HOME', 'The full path to the Gradle home')
```

Comment:
You might want to register this environment variable as an entry point (like https://github.com/colcon/colcon-core/blob/dff8696a2c17c9c353f919f7733395ca021a6c00/setup.cfg#L72) in order to make it appear when invoking `colcon --help`.